nservicebus/messaging/databus.md

```diff
@@ -68,7 +68,22 @@ snippet:MessageWithLargePayloadUsingConvention
 
 NServiceBus DataBus implementations currently behave differently with regard to cleanup of physical attachments used to transfer data properties. `FileShareDataBus` **does not** remove physical attachments once the message is gone. `AzureDataBus` **does** remove Azure storage blobs used for physical attachments once the message is gone.
 
+### Custom cleanup strategy
 
+NServiceBus cannot tell when a file should be deleted, as removing the file depends on your functional needs. There are a large number of scenarios where removing the file can cause a problem. 
```

Comment:
Avoid language with personal tone. See https://github.com/particular/docs.particular.net#language-preferences

How about:
NServiceBus does not remove the transferred message attachments once a message is delivered. Automatically removing these attachments can cause problems in many situations. For example: